nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/jquery/modal/jquery.modal.js

```diff
@@ -127,6 +128,10 @@
                 // check if the button should be disabled
                 if (isDisabled()) {
                     button.addClass('disabled-button');
+                    // remove keyup listener
+                    if (isDefinedAndNotNull(buttonConfig.handler) && isDefinedAndNotNull(buttonConfig.handler.keyup) && typeof buttonConfig.handler.keyup === 'function') {
+                        document.removeEventListener('keyup', buttonConfig.handler.keyup, true);
```

Comment:
Is there a reason we're not using jQuery for adding/removing the event listeners? This isn't a big deal but I'd prefer to remain consistent with the rest of the code base if possible. Does it have to do with the useCapture flag you've specified here?